test/processor/document_test.rb

```diff
@@ -384,6 +384,47 @@ def test_complex_field_conditional_inline
     assert_equal "ParagraphBefore Before Content After ParagraphAfter", text(with_true)
   end
 
+  def test_complex_field_without_end
```

Comment:
Can we rename this to make it clear that the field is actually ignored? Maybe `test_ignore_complex_field_spanning_multiple_paragraphs`